auv_sim/auv_sim_bridge/include/auv_sim_bridge/simulation_mock_ros.h

```diff
@@ -59,17 +49,25 @@ class SimulationMockROS {
     return true;
   }
 
-  void directDriveCallback(const auv_msgs::MotorCommand &msg) {
+  void drivePulseCallback(const auv_msgs::MotorCommand &msg) {
     const auto stamp = ros::Time::now();
+
+    // Remapping array: maps input channels to specific thruster indices
+    std::vector<int> channelRemap = {1, 7, 2, 5, 0, 6, 3, 4};
```

Comment:
does this reflect current mapping on main?

Reply:
I think so. This is the correct thruster mapping in simulation environment. Tested with publishing wrench commands.

Reply:
good!